Makefile

```diff
@@ -30,29 +30,29 @@ target:
 
 $(operator_bot):
 	cd python/operator && DDIT_VERSION=$(VERSION) $(PYTHON) setup.py sdist
-	rm -fr python/operator/dablchat_operator_bot.egg-info
+	rm -fr python/operator/daml-chat_operator_bot.egg-info
 	mkdir -p $(@D)
-	mv python/operator/dist/dablchat-operator-bot-$(VERSION).tar.gz $@
+	mv python/operator/dist/daml-chat-operator-bot-$(VERSION).tar.gz $@
 	rm -r python/operator/dist
 
 $(user_bot):
 	cd python/user && DDIT_VERSION=$(VERSION) $(PYTHON) setup.py sdist
-	rm -fr python/user/dablchat_user_bot.egg-info
+	rm -fr python/user/daml-chat_user_bot.egg-info
 	mkdir -p $(@D)
-	mv python/user/dist/dablchat-user-bot-$(VERSION).tar.gz $@
+	mv python/user/dist/daml-chat-user-bot-$(VERSION).tar.gz $@
 	rm -r python/user/dist
 
-$(ui): $(user_bot) 
+$(ui): $(user_bot)
 	yarn install
 	REACT_APP_ARCHIVE_BOT_HASH=$(shell sha256sum $(user_bot) | awk '{print $$1}') yarn build
-	zip -r dablchat-ui-$(VERSION).zip build
+	zip -r daml-chat-ui-$(VERSION).zip build
 	mkdir -p $(@D)
-	mv dablchat-ui-$(VERSION).zip $@
+	mv daml-chat-ui-$(VERSION).zip $@
 	rm -r build
 
 .PHONY: clean
 clean:
 	rm -fr \
-       python/operator/dablchat_operator_bot.egg-info python/operator/dist \
-       python/user/dablchat_user_bot.egg-info python/user/dist \
+       python/operator/daml-chat_operator_bot.egg-info python/operator/dist \
+       python/user/daml-chat_user_bot.egg-info python/user/dist \
```

Comment:
curious, do we want to mix snake and camel case here?

Reply:
seems like it is mixed anyway

Reply:
good catch, these files are supposed to be all snake case